.github/scripts/release-create-gh-release.sh

```diff
@@ -47,8 +47,8 @@ function publish_tag() {
 
 current_commit_msg=$(git show -s --format='%s' HEAD)
 
-if [[ "${current_commit_msg}" =~ Version[-[:space:]](v[[:digit:]]+.[[:digit:]]+.[[:digit:]]+) ]]; then
-    tag="${BASH_REMATCH[1]}"
+if [[ "${current_commit_msg}" =~ release/([[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+) ]]; then
+    tag="v${BASH_REMATCH[1]}"
```

Comment:
All changes in this file come from this PR https://github.com/MetaMask/metamask-extension/pull/36291, which was not included in 13.4.0, but was cherry-picked in 13.3.2.